proposals/NNNN-PerformInteractionMultipick.md

```diff
@@ -1,14 +1,14 @@
 
 # Perform Interaction Multipick
 
-* Proposal: [SDL-F111](F111-PerformInteractionMultipick.md)
+* Proposal: [SDL-NNNN](NNNN-PerformInteractionMultipick.md)
 * Author: [Michael Crimando](https://github.com/MichaelCrimando)
 * Status: **Awaiting review**
-* Impacted Platforms: [Core / iOS / Java Suite / RPC ]
+* Impacted Platforms: [Core / iOS / Java Suite / JavaScript / RPC ]

```

Comment:
`HMI` should be added here, based on your description in the `Impact on existing code` section.